frontend/src/components/Review/Review.tsx

```diff
@@ -55,50 +57,73 @@ const ReviewComponent = ({ review }: Props): ReactElement => {
   return (
     <Grid item>
       <Card className={root} variant="outlined">
-        <CardContent>
-          <Grid container spacing={2}>
-            <Grid item container justify="space-between">
-              <Grid container spacing={2}>
+        <Box minHeight="200px">
+          <CardContent>
+            <Grid container spacing={2}>
+              <Grid item container justify="space-between">
+                {overallRating !== null ? (
+                  <Grid container xs={10} spacing={2}>
+                    <Grid item>
+                      <HeartRating value={overallRating} readOnly />
+                    </Grid>
+                    <Grid item>
+                      <IconButton
+                        className={clsx(expand, {
+                          [expandOpen]: expanded,
+                        })}
+                        onClick={handleExpandClick}
+                        aria-expanded={expanded}
+                        aria-label="show more"
+                        size="small"
+                      >
+                        <ExpandMoreIcon />
+                      </IconButton>
+                    </Grid>
+                  </Grid>
+                ) : (
+                  <Grid item>
+                    <Typography>This review was collected by the Ithaca Tenants Union</Typography>
+                  </Grid>
+                )}
                 <Grid item>
-                  <HeartRating value={overallRating} readOnly />
+                  <Typography className={dateText}>{formattedDate}</Typography>
                 </Grid>
+              </Grid>
+              {overallRating !== null && (
                 <Grid item>
-                  <IconButton
-                    className={clsx(expand, {
-                      [expandOpen]: expanded,
-                    })}
-                    onClick={handleExpandClick}
-                    aria-expanded={expanded}
-                    aria-label="show more"
-                    size="small"
-                  >
-                    <ExpandMoreIcon />
-                  </IconButton>
+                  <Collapse in={expanded} timeout="auto" unmountOnExit>
+                    <CardContent>
+                      <DetailedRatings ratings={detailedRatings} />
+                    </CardContent>
+                  </Collapse>
                 </Grid>
+              )}
+              <Grid item container alignContent="center">
+                <Typography>
+                  {expandedText ? reviewText : reviewText.substring(0, 500)}
+                  {!expandedText && reviewText.length > 500 && '...'}
+                  {reviewText.length > 500 ? (
+                    <Button className={button} onClick={() => setExpandedText(!expandedText)}>
```

Comment:
Nice job with this! It's good that the button has hover effects. One thing we probably should do in the future for better UI is to fade the text behind the "Read More" button so that it's more obvious what the functionality is. 
<img width="715" alt="Screen Shot 2021-05-06 at 4 50 54 PM" src="https://user-images.githubusercontent.com/57203639/117363917-410d2600-ae8b-11eb-8a6e-e28a1d2bb7b6.png">